api/v2.0/swagger.yaml

```diff
@@ -2230,6 +2230,131 @@ paths:
           $ref: '#/responses/403'
         '500':
           $ref: '#/responses/500'
+  /usergroups:
+    get:
+      summary: Get all user groups information
+      description: Get all user groups information
+      tags:
+        - usergroup
+      responses:
+        '200':
+          description: Get user group successfully.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/UserGroup'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '500':
+          $ref: '#/responses/500'
+    post:
+      summary: Create user group
+      description: Create user group information
+      tags:
+        - usergroup
```

Comment:
use `operationId: createUserGroup` to this API

Reply:
include the `requestId` in the parameters?

Reply:
Updated